tools/pull-mdi-icons.js

```
@@ -51,32 +57,32 @@
 		} catch (e) {
-			log(chalk.red("Failed."));
+			log(red("Failed."));
 			return;
 		}
 		log(`Downloaded webfont ZIP ✔️`);
 		try {
-			await upstream.extractZip(
+			await extractZip(
 				webfontZipPath,
 				webfontExtractedZipPath,
 				`^MaterialDesign(Light)?-Webfont-${escapedVersion}\/(css\/materialdesignicons(-light)?\.min\.css|fonts\/materialdesignicons(-light)?-webfont\.woff2?)$`
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)